<h3>Trees, Fungi and Bacteria: Tripartite Metatranscriptomics of a Root Microbiome Responding to Soil Contamination</h3>
<h6>Gonzalez E., Brereton N.J.B., Marleau J., Guidi Nissim W., Page A., St-Arnaud M., Labrecque M., Joly S., Yergeau E. and Pitre F.E.</h6>
<br>
<h4>Figure 1. Total annotation</h4>
Annotation of the entire transcriptome assembly (including non-differentially expressed contigs). The proportion of each taxonomic grouping is defined by the number of unique transcripts, whereas the colour (if checked) represents the relative abundance (transcripts per million tpm) of transcripts in each taxon. Unknown contigs have no confident blast hit in the major protein repositories.
<h3><a target="_blank" href="http://htmlpreview.github.io/?https://github.com/gonzalezem/Tripartite_Metatranscriptomics_article/blob/master/Figure1/total%20annotation.html">GO TO FIGURE 1</a></h3>
<br>
<h4>Figure 2. Origin of differentially expressed contigs</h4>
 DE is estimated using EBseq with a posterior probability of differential expression (PPDE) ≥ 0.95. The proportion of each taxonomic grouping is defined by the number of unique transcripts, whereas the colour (if checked) represents the relative abundance (transcripts per million tpm) of transcripts in each taxon. Unknown contigs have no confident blast hit in the major protein repositories.
<h3><a target="_blank" href="http://htmlpreview.github.io/?https://github.com/gonzalezem/Tripartite_Metatranscriptomics_article/blob/master/Figure2/DE%20annotation%20total%20new%20tax%20c%20vs%20t.html">GO TO FIGURE 2</a></h3>
<br>
<h4>Figure 4A. Secondary annotation of fungal differentially expressed contigs</h4>
 Secondary annotation of each DE fungal contig illustrating alternative, equally valid annotation from other species (presented as genera for clarity). Genera with correspondences >20 are presented and coloured by DE direction (more abundant in contaminated roots = black; more abundant in non-contaminated roots = gold). 
<h3><a target="_blank" href="http://htmlpreview.github.io/?https://github.com/gonzalezem/Tripartite_Metatranscriptomics_article/blob/master/Figure4/Figure4A/Chord_Asco_Basidio_together_with_regulation.html">GO TO FIGURE 4A</a></h3>
<br>
<h4>Figure 4B. Origin of differentially expressed fungal contigs higher in contaminated trees</h4>
 DE is estimated using EBseq with a posterior probability of differential expression (PPDE) ≥ 0.95. The proportion of each taxonomic grouping is defined by the number of unique transcripts higher in contaminated conditions. The colour (if checked) represents the relative abundance (transcripts per million tpm) of transcripts in each taxon. Unknown contigs have no confident blast hit in the major protein repositories.
<h3><a target="_blank" href="http://htmlpreview.github.io/?https://github.com/gonzalezem/Tripartite_Metatranscriptomics_article/blob/master/Figure4/Figure4B/Fungal%20DE%20Contaminated.html">GO TO FIGURE 4B</a></h3>
<br>
<h4>Figure 4C. Origin of differentially expressed fungal contigs higher in non-contaminated trees</h4>
 DE is estimated using EBseq with a posterior probability of differential expression (PPDE) ≥ 0.95. The proportion of each taxonomic grouping is defined by the number of unique transcripts higher in non-contaminated conditions. The colour (if checked) represents the relative abundance (transcripts per million tpm) of transcripts in each taxon. Unknown contigs have no confident blast hit in the major protein repositories.
<h3><a target="_blank" href="http://htmlpreview.github.io/?https://github.com/gonzalezem/Tripartite_Metatranscriptomics_article/blob/master/Figure4/Figure4C/Fungal%20DE%20NonContaminated.html">GO TO FIGURE 4C</a></h3>
<br>
<h4>Figure 6. Bacterial contigs, total and differentially expressed (DE) transcript origin</h4>
 Krona graphs represent A) total annotation of bacterial transcripts (including non-DE) and B) annotation of DE bacterial transcripts. EBseq was used to estimate posterior probability of differential expression (PPDE) ≥ 0.95. The proportion of each taxonomic grouping is defined by the number of unique transcripts, whereas the colour (if checked) represents the relative abundance (transcripts per million tpm) of transcripts in each taxon.
<h3><a target="_blank" href="http://htmlpreview.github.io/?https://github.com/gonzalezem/Tripartite_Metatranscriptomics_article/blob/master/Figure6/Total%20Bacteria.html">GO TO FIGURE 6A</a></h3>
<h3><a target="_blank" href="http://htmlpreview.github.io/?https://github.com/gonzalezem/Tripartite_Metatranscriptomics_article/blob/master/Figure6/Bacteria%20DE.html">GO TO FIGURE 6B</a></h3>


